lms/models/event.py

```diff
@@ -20,7 +20,7 @@ class Type(str, Enum):
         GRADE = "grade"
         ERROR_CODE = "error_code"
 
-    id = sa.Column(sa.Integer, autoincrement=True, primary_key=True)
+    id = mapped_column(sa.Integer, autoincrement=True, primary_key=True)
```

Comment:
These fixes errors elsewhere, they seem to provide better type inference via SQLAlchemy.